docs/web/user_guide.md

```diff
@@ -171,7 +171,7 @@ optional arguments:
   -f CONFIG_DIRECTORY, --config-directory CONFIG_DIRECTORY
                         Directory where CodeChecker server should read server-
                         specific configuration (such as authentication
-                        settings, and SSL certificates) from.
+                        settings, and TLS/SSL certificates) from.
                         (default: /home/<username>/.codechecker)
```

Comment:
These texts should be in synch with the contents of the Python script that generates the help when the user says `--help`. Please add the same change to

https://github.com/Ericsson/codechecker/blob/01a042ce6442f66d007fbca750951f6f904d350c/web/server/codechecker_server/cmd/server.py#L89-L97

Then we can merge! :smile: